.github/workflows/ci.yaml

```
@@ -11,15 +11,14 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/ci/containers/base:latest
```

I suggest to use

```suggestion
      image: registry.opensuse.org/opensuse/tumbleweed
```

as I don't see any relation to devel:openQA and we shouldn't need that. I also just created https://build.opensuse.org/package/show/home:okurz:container/logwarn so in some minutes you can use registry.opensuse.org/home/okurz/container/containers/tumbleweed:logwarn

---

Well this **is** openqa-logwarn ;-) but a dedicated container is even better